components/gitpod-protocol/src/protocol.ts

```diff
@@ -323,6 +320,21 @@ export namespace Identity {
     }
 }
 
+export interface Email {
+    address: string;
+    type: string;
```

Comment:
We definitely should make `type` typesafe by having a `export type EmailType = "commit" | "block_pass"` (or similar).

Reply:
👍  `type` is now `COMMIT | OTHER` since the block_pass logic is handled in `user-service.ts`.